app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -818,6 +863,10 @@ private var progressBar =
         UiSelector().resourceId("$packageName:id/mozac_browser_toolbar_progress")
     )
 
+private val suggestedLoginsButton = mDevice.findObject(UiSelector().resourceId("$packageName:id/loginSelectBar"))
+
+private val suggestedLoginsView = mDevice.findObject(UiSelector().resourceId("$packageName:id/loginSelectBar"))
```

Comment:
Nit: This is the same as suggestedLoginsButton... can't we use it in general, not button or view, so that we don't have to duplicate this line?

Reply:
Sure, done !
Thanks ! ☺️